### ISSUE TITLE
Nicht unterstützte ActiveSync Funktion

### ISSUE THREAD

**Issue by Matze0811** [tine20/tine20#7170]
Hallo Leute, ist zwar offensichtlich nur eine Kleinigkeit, die die Funktion von tine2.0 nicht beeinflusst, ich bekomme halt trotzdem diese Fehlermeldung: matthias.pfahl - 2020-04-08T11:02:05+00:00 CRIT (2): Syncroton_Server::_handlePost::110 command not supported: ResolveRecipients
in die Logs. Ich verwende als AS-Client ein Samsung Galaxy Note 8 und die verwendete Tine20 Version ist: 2020.03.1

Ich hab mal schnell gegoogelt und herausgefunden das das eine Funktion von Exchange ist um die Free und Busy Zeiten von Benutzern zu syncen. Vielleicht könnt Ihr Euch das ja mal ansehen.

Frohe Ostern und bleibt gesund!

**Comment by pschuele** [comment]
hallo Matze,
danke für die info. ich packe das mal auf "feature request". allerdings finde ich CRIT auch etwas hart als logmeldung, denke, das sollten wir auf alle fälle mal z.b. auf NOTICE runterstellen.

**Comment by Matze0811** [comment]
Ja, feature request ist natürlich in Ordnung! Hatte mich schon gewundert, was an der Meldung "kritisch" sein soll, da alle Funktionen wohl soweit laufen!
